public/client-v2/src/components/FrontsEdit/Edit.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react';
+import Fronts from './Fronts';
+
+type Props = {
+  match: Object
```

Comment:
Could we type this with the `Match` type from `redux-router`. Object will just have an `any` type for all string keys.